src/main/java/uk/gov/ons/census/action/schedule/ActionRuleTriggerer.java

```diff
@@ -24,7 +27,12 @@ public void triggerActionRules() {
         actionRuleRepo.findByTriggerDateTimeBeforeAndHasTriggeredIsFalse(OffsetDateTime.now());
 
     for (ActionRule triggeredActionRule : triggeredActionRules) {
-      actionRuleProcessor.createScheduledActions(triggeredActionRule);
+      try {
+        actionRuleProcessor.createScheduledActions(triggeredActionRule);
+      } catch (Exception e) {
+        log.with("action_rule_id", triggeredActionRule.getId())
+            .error("Stop putting untested SQL into production", e);
```

Comment:
msg could be a little more professional/dull.  
Odds are the person reading this msg didn't put bad SQL into production.  
Also we might get this error in none production environments.